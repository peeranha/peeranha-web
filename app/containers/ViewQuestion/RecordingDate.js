import React from 'react';
import PropTypes from 'prop-types';
import { getFormattedDate } from 'utils/datetime';

const RecordingDate = props => (
  <p className="recording-date">{getFormattedDate(props.postTime)}</p>
);

RecordingDate.propTypes = {
  postTime: PropTypes.number,
};

export default RecordingDate;
