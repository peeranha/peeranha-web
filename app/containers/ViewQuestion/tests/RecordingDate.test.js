import RecordingDate from '../RecordingDate';

jest.mock('utils/datetime');

describe('RecordingDate', () => {
  it('test', () => {
    const props = {
      postTime: 1542624898950,
    };
    expect(RecordingDate(props)).toMatchSnapshot();
  });
});
