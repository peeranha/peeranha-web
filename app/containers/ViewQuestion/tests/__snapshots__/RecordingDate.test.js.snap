// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecordingDate test 1`] = `
<p
  className="recording-date"
/>
`;
